src/core/Array.cpp

```diff
@@ -158,6 +158,35 @@ void Array::sort_custom(Object *obj, const String &func) {
 	godot::api->godot_array_sort_custom(&_godot_array, (godot_object *)obj, (godot_string *)&func);
 }
 
+int Array::bsearch(const Variant &value, const bool before) {
+	return godot::api->godot_array_bsearch(&_godot_array, (godot_variant *)&value, before);
+}
+
+int Array::bsearch_custom(const Variant &value, const Object &obj,
```

Comment:
I have no idea how C++ handles this with inheritance and all, but I would expect a `const Object *` here instead of a reference.

Reply:
Fixed!